src/www/ui/page/AdminLicenseCandidate.php

```diff
@@ -61,6 +67,14 @@ function __construct()
    */
   protected function handle(Request $request)
   {
+    $action = $request->get('action');
```

Comment:
later there is a switch-case on `$request->get('do')` which determines the action. Maybe it would be good to use this field also here.